ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -234,7 +239,7 @@ const EthOverview = ({ className, showAddress }) => {
                   color={IconColor.primaryInverse}
                 />
               }
-              disabled={!isBuyableChain}
+              disabled={!isBuyableChain || !signingEnabled}
```

Comment:
According to my previous comment, you can revert this since this button just redirects to the portfolio
```suggestion
              disabled={!isBuyableChain}
```